recipes/physiocurve/meta.yaml

```diff
@@ -0,0 +1,80 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Ping!

Reply:
Sorry, fixed in the last commit

Reply:
Been there, done that, and I wrote that example!

BTW, try grayskull next time you submit a recipe. It will make the recipe almost "merge ready."

Reply:
Nope. That did not change :-(

We hope to fix that soon though. Pinging @marcelotduarte and @ForgottenProgramme here just for awareness b/c they are involved in that effort.